kobocat-template/templates/legacy_survey.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 {% load static %}
 
-{% with legacy_survey_url="?todo" %}
+{% with legacy_survey_url="https://ee.kobotoolbox.org/single/XyEzJRoc" %}
```

Comment:
i'm checking with Tino to see if this should be something we configure for only our instances or if all self-hosters should see this survey as well
https://www.flowdock.com/app/kobotoolbox/ux-ui/threads/s7ZEfrky3wE9cWEDJ8pAsA74G5o

Reply:
Doesn't matter